File: algorithm笔记/冒泡排序/bubble.js
function bubbleSort(list_data) {
	let sign = true;
	for (let index = 0; index < list_data.length - 1; index++) {
		let i, j;
		if (list_data[index] > list_data[index + 1]) {
			i = list_data[index];
			j = list_data[index + 1];
			list_data[index] = j;
			list_data[index + 1] = i;
			sign = false;
		}
	}
	if (!sign) bubbleSort(list_data);
}

File: algorithm笔记/冒泡排序/bubble.py
def bubbleSort(list_data):
	length = len(list_data)
	sign = True
	index = 0
	while index < length - 1:
		if(list_data[index] > list_data[index + 1]):
			i = list_data[index]
			j = list_data[index + 1]
			list_data[index] = j
			list_data[index + 1] = i
			sign = False
		index += 1
	if sign == False:
		return bubbleSort(list_data)